pkg/render/monitor.go

```diff
@@ -308,6 +311,29 @@ func (mc *monitorComponent) serviceMonitorElasicsearch() *monitoringv1.ServiceMo
 	}
 }
 
+// TODO Remove this object after we drop support for v3.8.
```

Comment:
Could you include a note like "This is to delete a service that had been released with a typo in the name", just so it is obvious why we're deleting it and what was wrong with it?